### PR TITLE
Docs: Specify process for adding a world maintainer to an existing world

### PR DESCRIPTION
**Pull request by NewSoupVi**
ESL, feel free to make suggestions for how to phrase this better

### CODE REVIEW

File: docs/world maintainer.md
Comment (Exempt-Medic) [inline docs/world maintainer.md:36]: I don't know if you'd want to account for this, but what if somebody is multiple of these things? Does a new Witness or ALttP or SoE etc. maintainer only require one approval?

**Review by Exempt-Medic — APPROVED** [review]
The content looks good to me. One small wording comment and one policy comment.

**Review by ScipioWright — APPROVED** [review]
LGTM

File: docs/world maintainer.md
Comment (NewSoupVi) [reply docs/world maintainer.md:36]: Good point - I think someone opening a PR to be added to Witness would only require my approval, but we could make it such that it requires a different core maintainer as well

Comment (black-sliver) [reply docs/world maintainer.md:36]: Interesting. I think it should probably not be necessary because the trust boundary is the same if V approves it for witness or if V approves it for another game.

**Review by black-sliver — APPROVED** [review]
Gonna approve as-is; see the comment I left.

**Pull request merged**
